default_warehouse_from_sale_team/models/sales_team.py

```diff
@@ -88,3 +88,27 @@ def default_get(self, fields_list):
                 {name: warehouse_id for name in names_list
                  if defaults.get(name)})
         return defaults
+
+    @api.v7
+    def read(self, cr, user, ids, fields=None, context=None,
+             load='_classic_read'):
+        """This method is overwrite because we need to propagate SUPERUSER_ID
+        when picking are chained in another warehouse without access to read"""
+        if self.pool.get('ir.model.access').check_groups(
+            cr, user, 'default_warehouse_from_sale_team.'
+                'group_limited_default_warehouse_sp'):
+            return super(WarehouseDefault, self).read(
+                cr, SUPERUSER_ID, ids, fields=fields, context=context,
+                load=load)
+        return super(WarehouseDefault, self).read(
+            cr, user, ids, fields=fields, context=context, load=load)
+
+    @api.v8
+    def read(self, fields=None, load='_classic_read'):
+        """This method is overwrite because we need to propagate SUPERUSER_ID
+        when picking are chained in another warehouse without access to read"""
+        if self.env['ir.model.access'].check_groups(
```

Comment:
@JulioSerna 

in v8 why do you not use self.env.user.has_group('default_warehouse_from_sale_team.group_limited_default_warehouse_sp')